app/[locale]/forms/layout.tsx

```diff
@@ -0,0 +1,14 @@
+import { type ReactNode } from 'react';
+
+import { Toaster } from '~/components/ui/toaster';
+
+function layout({ children }: { children: ReactNode }) {
+  return (
+    <>
+      {children}
+      <Toaster />
+    </>
+  );
+}
+
+export default layout;
```

Comment:
No need to export in a different line, just do `export default function Layout`. Also, you should put a WIP `page.tsx` here instead of making a layout, since there will be a default forms landing page in the future.